lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -437,14 +437,24 @@ class ByteCodeBufferBuilder
             V3.value = buildDateHash;
             V4.value = buildTimeHash;
         }
-        else
+        else if (!GenerateLibraryByteCode())
         {
             auto guidDWORDs = (DWORD*)(&byteCodeCacheReleaseFileVersion);
             V1.value = guidDWORDs[0];
             V2.value = guidDWORDs[1];
             V3.value = guidDWORDs[2];
             V4.value = guidDWORDs[3];
         }
+        else
+        {
+            // To keep consistent library code between Chakra.dll and ChakraCore.dll, use a fixed version.
+            // This goes hand in hand with the bytecode verification unit tests.
+            V1.value = 0;
+            V2.value = 0;
+            V3.value = 0;
+            V4.value = 0;
```

Comment:
For the library code it doesn't really matter, as long as the unit tests catch the incorrectness. We usually catch this in unit tests when we fail unit tests, but we don't always catch them right now. E.g. the jshost/ch inconsistency fixed in this change. The unit tests are guaranteed to catch the bytecode changes.